lib/helpers/isTruthy.ts

```diff
@@ -0,0 +1,3 @@
+export default function<T>(t: T | undefined | null | void | false | number): t is T {
```

Comment:
Either the name is not ideal or the types. `t is T` means that the function is a type predicate: https://www.typescriptlang.org/docs/handbook/advanced-types.html#using-type-predicates

So basically it just allows you to write a function that safely asserts some type (as opposed to using `t as T` or discrimination - `'prop' in T ? ... : ...`).

 ```suggestion
export default function<T>(t: T) {
```

Reply:
I understand the problem now. Let's leave it as is if it works.